views/js/qtiItem/helper/modalFeedback.js

```diff
@@ -0,0 +1,175 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+define([
+    'jquery',
+    'lodash',
+    'taoQtiItem/qtiItem/helper/pci',
+    'taoQtiItem/qtiItem/helper/container'
+], function($, _, pci, containerHelper) {
+    'use strict';
+
+    /**
+     * Provide the feedbackMessage signature to check if the feedback contents should be considered equals
+     *
+     * @param {type} feedback
+     * @returns {String}
+     */
+    var getFeedbackMessageSignature = function getFeedbackMessageSignature(feedback) {
+        return ('' + feedback.body() + feedback.attr('title')).toLowerCase().trim().replace(/x-tao-[a-zA-Z0-9\-._\s]*/g, '');
+    };
+
+    /**
+     * Extract the display information for an interaction-related feedback
+     *
+     * @private
+     * @param {Object} interaction - a qti interaction object
+     * @returns {Object} Object containing useful display information
+     */
+    var extractDisplayInfo = function extractDisplayInfo(interaction) {
+
+        var $interactionContainer = interaction.getContainer();
+        var responseIdentifier = interaction.attr('responseIdentifier');
+        var messageGroupId, $displayContainer;
+
+        if (interaction.is('inlineInteraction')) {
+            $displayContainer = $interactionContainer.closest('[class*=" col-"], [class^="col-"]');
+            messageGroupId = $displayContainer.attr('data-messageGroupId');
+            if (!messageGroupId) {
+                //generate a messageFromId
+                messageGroupId = _.uniqueId('inline_message_group_');
+                $displayContainer.attr('data-messageGroupId', messageGroupId);
+            }
+        } else {
+            messageGroupId = responseIdentifier;
+            $displayContainer = $interactionContainer;
+        }
+
+        return {
+            responseIdentifier: responseIdentifier,
+            interactionContainer: $interactionContainer,
+            displayContainer: $displayContainer,
+            messageGroupId: messageGroupId,
+            order: -1
+        };
+    };
+
+    /**
+     * Get interaction display information sorted in the order of appearance within the item
+     *
+     * @param {Object} item
+     * @returns {Array}
+     */
+    var getInteractionsDisplayInfo = function getInteractionsDisplayInfo(item) {
+
+        var interactionsDisplayInfo = [];
+        var $itemContainer = item.getContainer();
+        var interactionOrder = 0;
+
+        //extract all interaction related information needed to display their
+        _.each(item.getComposingElements(), function (element) {
```

Comment:
Please use `_.forEach` (we want to limit the use of aliases to facilitate the migration to the next version of lodash)